syz-manager/manager.go

```diff
@@ -50,9 +51,10 @@ import (
 )
 
 var (
-	flagConfig = flag.String("config", "", "configuration file")
-	flagDebug  = flag.Bool("debug", false, "dump all VM output to console")
-	flagBench  = flag.String("bench", "", "write execution statistics into this file periodically")
+	flagConfig      = flag.String("config", "", "configuration file")
+	flagDebug       = flag.Bool("debug", false, "dump all VM output to console")
+	flagBench       = flag.String("bench", "", "write execution statistics into this file periodically")
+	flagHeapProfile = flag.String("heap_profile", "", "periodically write pprof heap profile into this file")
```

Comment:
Can we move this either to pkg/tool, or to testbed? manager.go is huge and this is quite generic low-level functionality.

Reply:
By "move to testbed" -- you mean, make http requests to `/debug/pprof/heap`?

`pkg/tool` might be problematic. We don't use it in `syz-manager` and in its current form it doesn't add much value to it.

Reply:
> By "move to testbed" -- you mean, make http requests to /debug/pprof/heap?

Yes.

> pkg/tool might be problematic. We don't use it in syz-manager and in its current form it doesn't add much value to it.

it can be a separate function in pkg/tool that does just it.